<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Benutzerhandbuch</class>
 <widget class="QDialog" name="Benutzerhandbuch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="MSchliessen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="JDieWettkaempfeZeitWettkaempfe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Die Wettkämpfe 
 Zeit-Wettkämpfe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HDasLernenAnfaengerUebungen">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Das Lernen 
 Anfänger-Übungen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GDasHauptmenue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Das Hauptmenü</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FUmgangmitderBildschirmtastatur">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Umgang mit der 
Bildschirmtastatur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ELernvorgang">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Lernvorgang</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AInhaltsangabe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Inhaltsangabe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BVorwort">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorwort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IDasLernenFortgeschrittenenUebungen">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Das Lernen
 Fortgeschrittenen-Übungen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="KDieWettkaempfeMultiplayerWettkampf">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Die Wettkämpfe 
 Multiplayer-Wettkampf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DBenutzerkontofuehrung">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Benutzerkontoführung</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CEinfuehrungTypo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Einführung Typo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AAVollstaendigesHandbuch">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vollständiges Handbuch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LStatistik">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistik</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>21</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>21</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
